src/runtime/lpython_intrinsic_numpy.py

```diff
@@ -276,24 +274,24 @@ def arctan(x: f32) -> f32:
 @overload
 @vectorize
 def degrees(x: f64) -> f64:
-    return x*180/pi_64
+    return x*180.0/pi_64
 
 @overload
 @vectorize
 def degrees(x: f32) -> f32:
-    return x*180/pi_32
+    return x*f32(f32(180.0)/pi_32)
 
 ########## radians ##########
 
 @overload
 @vectorize
 def radians(x: f64) -> f64:
-    return x*pi_64/180
+    return x*pi_64/180.0
 
 @overload
 @vectorize
 def radians(x: f32) -> f32:
-    return x*pi_32/180
+    return x*f32(pi_32/f32(180.0))
```

Comment:
Can this be written just as:
```
x*pi_32/f32(180)
```
? Or will that not work.

Reply:
Actually `/` operator always returns `f64` type. This is because I think CPython always returns `float` with `/` operator. `float` is equivalent to `f64` so we need to manually cast it to `f32`.

Reply:
This was happening implicitly but now got visibility when implicit casting is turned off.

Reply:
I see. Yes, Python returns float for /.

But we currently do `f32 * f32 -> f32`, that is, multiplication keeps the kind of the floating point. So why not also do `f32 / f32 -> f32` and make the division keep the kind? 

In Python it is simply `float * float -> float` and `float / float -> float`.

Reply:
Makes sense. I am +1 on this.